open-apparel-registry/open-apparel-registry
# Add warning about inexact facility coordinates

## Overview

Sometimes facilities are geolocated incorrectly or are unable to be
geolocated. In that situation, admins will manually update the location;
however, we need to alert the users that the facility coordinates are
known to be inaccurate. In the Django admin, users are now also able to
note that the facility is known to be inaccurate. When coordinates have
been labeled as inexact, we add text to the facility details noting that
the location is known to be inexact.

Connects #1468 

## Demo

<img width="353" alt="Screen Shot 2021-10-07 at 1 23 37 PM" src="https://user-images.githubusercontent.com/21046714/136983855-199394e5-a3d8-4c0c-97d3-3a9b943406d0.png">

## Testing Instructions

* Run `./scripts/manage migrate` and then `./scripts/server`
* Navigate to a facility in the Django admin, and check the box 'has inexact coordinates' and save.
* Navigate to the facility details page; under the GPS coordinates, there should be a message warning that the coordinates are inexact. Text should match the text in the issue. 
* View a different facility's details page, and confirm that it does not include the warning.

## Checklist
